classes/ft-ps-client.php

```diff
@@ -102,7 +102,7 @@ function init_premium_support() {
 				// Ping server for response
 				if ( $request = wp_remote_post( $this->server_url, array( 'body' => $body, 'timeout' => 20 ) ) ) {
 
-					if ( isset( $request['response']['code'] ) && 200 == $request['response']['code'] ) {
+					if ( ! is_wp_error( $request ) && isset( $request['response']['code'] ) && 200 == $request['response']['code'] ) {
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Expected 0 spaces after opening bracket; 1 found
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 134 characters
* Blank line found at start of control structure

<!-- Comment by [SideCI](https://sideci.com) -->